### PR TITLE
fix: update env.examples

### PR DESCRIPTION
**Pull request by gerimate**
Fix and add documentation in `.env.example` files

- dyrectorio/.env.example
- dyrectorio/web/crux/.env.example
- dyrectorio/web/crux-ui/.env.example
- dyrectorio/golang/cmd/crane.env.example
- dyrectorio/golang/cmd/dagent.env.example

### CODE REVIEW

**Review by robot9706 — APPROVED** [review]

File: .env.example [hunk 1]
```diff
@@ -1,34 +1,75 @@
...
+# meaning that in the order above they contain each other
+# Example: 'warning' contains 'error'
 LOG_LEVEL=debug
 
-# Database passwords
+## Database passwords
+
+# This value is the password to crux's database
 CRUX_POSTGRES_PASSWORD=Random_Generated_String
+# This value is the password to kratos's database
```
Comment (m8vago) [inline .env.example:36]: kratos'

**Review by m8vago — APPROVED** [review]

File: .env.example [hunk 2]
```diff
@@ -1,34 +1,75 @@
-# Docker settings
+## Docker settings
+
+# Traefik requires this file to be able to route the requests to the containers
 DOCKER_SOCKET=/var/run/docker.sock
 
-# Domain settings
+## Domain settings
```
Comment (nandor-magyar) [inline .env.example:6]: If this is a higher category `domain` then the following should all be related to domain configuration and they are not. Shouldn't it be General or global? Is this even necessary?

File: .env.example [hunk 3]
```diff
@@ -1,34 +1,75 @@
...
+# E-mail sender name for dyrector.io invitation links,
+# password resets and others
 FROM_NAME=dyrector.io
 
-# Recaptcha secrets
-# In case you don't want to use ReCAPTCHA set DISABLE_RECAPTCHA to true
+## ReCAPTCHA secrets
+
+# In case you don't want to use ReCAPTCHA set DISABLE_RECAPTCHA to true.
+# Highly recommended to set DISABLE_RECAPTCHA to false
```
Comment (nandor-magyar) [inline .env.example:70]: Default value is not completely obvious, about the comments: Default: false => it is on.

File: golang/cmd/crane/.env.example [hunk 4]
```diff
@@ -14,15 +14,24 @@ DEBUG_UPDATE_ALWAYS=false
 DEBUG_UPDATE_USE_CONTAINERS=true
 DEFAULT_REGISTRY=index.docker.io
 
-# crane specific options
+# Crane specific options
+# Under obsoletion
 CRANE_GEN_TCP_INGRESS_MAP=
```
Comment (nandor-magyar) [inline golang/cmd/crane/.env.example:19]: Safe to remove, not used in the code, not parsed.

File: golang/cmd/crane/.env.example [hunk 5]
```diff
@@ -14,15 +14,24 @@ DEBUG_UPDATE_ALWAYS=false
...
+# Put 'true' to use in-cluster auth
 CRANE_IN_CLUSTER=true
```
Comment (nandor-magyar) [inline golang/cmd/crane/.env.example:21]: Default is false, kubeconfig will be search at default paths.

File: golang/cmd/crane/.env.example [hunk 6]
```diff
@@ -14,15 +14,24 @@ DEBUG_UPDATE_ALWAYS=false
...
+# Kube
```
Comment (nandor-magyar) [inline golang/cmd/crane/.env.example:22]: The duration amount that for a kubernetes API request to complete.

File: golang/cmd/crane/.env.example [hunk 7]
```diff
@@ -14,15 +14,24 @@ DEBUG_UPDATE_ALWAYS=false
...
+# The key/label name for audit purposes
 KEY_ISSUER=co.dyrector.io/issuer
+# The "kubectl" configuration location
 KUBECONFIG=
+# Timeouts used in tests, no effect on deployment
 TEST_TIMEOUT=15s
+# For injecting SecretPrivateKey
 SECRET_NAME=dyrectorio-secret
 SECRET_NAMESPACE=dyrectorio
 CRANE_IN_CLUSTER=false
```
Comment (nandor-magyar) [inline golang/cmd/crane/.env.example:37]: Twice in the same file.

File: golang/cmd/dagent/.env.example [hunk 8]
```diff
@@ -18,18 +18,30 @@ AGENT_CONTAINER_NAME=dagent
...
+# This should match the mount path that is
+# the root of configurations and containers
 DATA_MOUNT_PATH=/srv/dagent
 DEFAULT_TAG=latest
 DEFAULT_TIMEOUT=5s
 GRPC_KEEPALIVE=60s
+# Path of 'docker.sock' or other local/remote
+# address where we can communicate with docker
 HOST_DOCKER_SOCK_PATH=/var/run/docker.sock
+HOST_MOUNT_PATH=/srv/dagent
```
Comment (nandor-magyar) [inline golang/cmd/dagent/.env.example:30]: Non-existent variable.

File: golang/cmd/dagent/.env.example [hunk 9]
```diff
@@ -18,18 +18,30 @@ AGENT_CONTAINER_NAME=dagent
...
+# Containers mount path default
 INTERNAL_MOUNT_PATH=/srv/dagent
+# Loglines to skip.
```
Comment (nandor-magyar) [inline golang/cmd/dagent/.env.example:33]: addition: if not defined on the request

**Review by nandor-magyar — COMMENTED** [review]
In some cases commenting the existing default values help a lot & makes it easier to ingest, those I miss but also commented where I've found it helpful.

File: .env.example (hunk 2 above)
Comment (gerimate) [reply .env.example:6]: @m8vago thoughts?

File: golang/cmd/crane/.env.example (hunk 7 above)
Comment (gerimate) [reply golang/cmd/crane/.env.example:37]: Can't find it

File: golang/cmd/crane/.env.example (hunk 5 above)
Comment (nandor-magyar) [reply golang/cmd/crane/.env.example:21]: This is the CRANE_IN_CLUSTER the same. @gerimate

**Review by nandor-magyar — APPROVED** [review]

**Pull request merged**